central/config/datastore/datastore_test.go

```diff
@@ -50,24 +50,94 @@ func (s *configDataStoreTestSuite) TearDownTest() {
 	s.mockCtrl.Finish()
 }
 
+var (
+	sampleConfig = &storage.Config{
+		PublicConfig: &storage.PublicConfig{
+			LoginNotice: &storage.LoginNotice{
+				Enabled: false,
+				Text: "You step onto the road, and if you don't keep your feet, " +
+					"there's no knowing where you might be swept off to.",
+			},
+			Header: &storage.BannerConfig{
+				Enabled: false,
+				Text: "Home is behind, the world ahead, and there " +
+					"are many paths to tread through shadows to the edge of night, " +
+					"until the stars are all alight.",
+				Size_:           10,
+				Color:           "0x88bbff",
+				BackgroundColor: "0x0000ff",
+			},
+			Footer: &storage.BannerConfig{
+				Enabled:         false,
+				Text:            "All's well that ends better.",
+				Size_:           10,
+				Color:           "0x88bbff",
+				BackgroundColor: "0x0000ff",
+			},
+			Telemetry: nil,
+		},
+		PrivateConfig: &storage.PrivateConfig{
+			AlertRetention:                      nil,
+			ImageRetentionDurationDays:          7,
+			ExpiredVulnReqRetentionDurationDays: 7,
+			DecommissionedClusterRetention:      nil,
+			ReportRetentionConfig:               nil,
+			VulnerabilityDeferralConfig:         nil,
+		},
+	}
+)
+
+func (s *configDataStoreTestSuite) TestAllowsGetPublic() {
+	s.storage.EXPECT().Get(gomock.Any()).Return(sampleConfig, true, nil).Times(1)
```

Comment:
nitpick: Maybe we should also add `getPublicConfigCache().Purge()`, just to be on the safe side and that the order of execution does not influence tests.